test/js/specs/happy.js

```diff
@@ -409,4 +409,184 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
       })
     })
   })
-})
\ No newline at end of file
+  describe('PROOF OF ADDRESS', async () => {
+
+    const goToPoADocumentSelectionScreen = async () => {
+      driver.get(localhostUrl + `?poa=true&?async=false&useWebcam=false`)
+      welcome.primaryBtn.click()
+      poaIntro.clickStartVerificationButton()
+    }
+
+    const uploadFileAndClickConfirmButton = async (fileName) => {
+      documentUpload.getUploadInput()
+      documentUpload.upload(fileName)
+      documentUploadConfirmation.confirmBtn.click()
+    }
+
+    it('should verify UI elements of PoA Intro screen', async () => {
+      const poaIntroCopy = poaIntro.copy()
+      driver.get(localhostUrl + `?poa=true`)
+      welcome.primaryBtn.click()
+      poaIntro.verifyTitle('Let’s verify your UK address')
+      poaIntro.verifyRequirementsHeader(poaIntroCopy)
+      poaIntro.verifyFirstRequirement('Shows your current address')
+      poaIntro.verifySecondRequirement('Matches the address you used on signup')
+      poaIntro.verifyThirdRequirement('Is your most recent document')
+      poaIntro.verifyStartVerificationButton(poaIntroCopy)
+    })
+
+    it('should verify UI elements of PoA Document Selection screen', async () => {
+      const poaDocumentSelectionCopy = poaDocumentSelection.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.verifyTitle('Select a UK document')
+      poaDocumentSelection.verifySubtitle(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsBankCell(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsUtilityBillCell(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsCouncilTaxLetter(poaDocumentSelectionCopy)
+      poaDocumentSelection.verifyElementsBenefitsLetter(poaDocumentSelectionCopy)
+    })
+
+    it('should verify UI elements of PoA Guidance for Bank Statement', async () => {
+      const poaGuidanceCopy = poaDocumentSelection.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnBankIcon()
+      poaGuidance.verifyUIElementsForBankStatementGuidanceScreen(poaGuidanceCopy)
+      poaGuidance.verifySubtitle('Must be issued in the last 3 months')
+      poaGuidance.verifyFullNameText('Full name')
```

Comment:
No need for this repetition. You can wrap `verifyCurrentText`, `verifyAddressText`, `verifyIssueDateText`, `verifySummaryPeriodText` into one function inside `PoaGuidance` class, then call it for each one of the documents

Reply:
Addressed in: ab14745